Source/RenderStream/Public/RenderStreamLink.h

```diff
@@ -28,7 +28,7 @@ typedef uint64_t VkDeviceSize;
 typedef struct VkSemaphore_T* VkSemaphore;
 
 #define RS_PLUGIN_NAME "RenderStream-UE"
-#define RS_PLUGIN_VERSION "RS2.0-UE5.4-v1"
+#define RS_PLUGIN_VERSION "RS2.0-UE5.5-v1"
```

Comment:
Quick q: should it be v1 or v0?

Reply:
Looking back at old versions it looks like it should be v0, I will change it